pkg/controller/sparkapplication/controller_test.go

```diff
@@ -823,7 +823,7 @@ func TestSyncSparkApp_SubmissionSuccess(t *testing.T) {
 	}
 }
 
-func TestSyncSparkApplication_ExecutingState(t *testing.T) {
+func TestSyncSparkApplication_ExecutorState(t *testing.T) {
```

Comment:
We can also add a quick test here for Unknown

Reply:
Done.